src/views/ContactReport.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <b-container>
+    <h1>Contact Report</h1>
```

Comment:
Please add more margin between the headline and the filter.

Reply:
I think this should be defined global over all views. Added global class `title`